packages/viewer/src/components/Shared/Source.svelte

```diff
@@ -13,8 +15,12 @@
   {#if showMarkdown}Hide{:else}Show{/if} markdown
 </button>
 {#if showMarkdown}
-  <h4>π-Base Markdown</h4>
-  <pre>{source}</pre>
-  <hr />
-  <ExtMarkdown {source} />
+  {#if internal}
+    <h4>π-Base Markdown</h4>
+    <pre>{source}</pre>
+    <hr />
```

Comment:
```suggestion
    {#if external}
      <hr />
    {/if}
```

So the bar only shows if both things appear